gui/hud.gui

```diff
@@ -78,7 +78,10 @@ widget = {
 	notifications_alerts = {
```

Comment:
So you see there's a small gap between the hud and the mana tooltip.
If your mouse goes there, the mana tooltip closes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120375340-bd304780-c32b-11eb-8da6-994c3251bdb8.png)

</details>

Reply:
Report to paradox? It's vanilla interface.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/120464853-0fae4a00-c3a6-11eb-95aa-45fc5b21667f.png)

</details>

Reply:
> Report to paradox? It's a part of vanilla interface.

Mana is further to the left than the gold.

Reply:
> Mana is further to the left than the gold.

It uses the same interface as the gold

Reply:
> It uses the same interface as the gold

Mana is further to the left than the gold. I compared it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120517819-e57a7d80-c3e1-11eb-803f-52894e9252b7.png)
![image](https://user-images.githubusercontent.com/46384992/120517823-e8756e00-c3e1-11eb-817b-fbffe828ec33.png)

</details>

Reply:
Better? 79c6fb1dc